sycl/test-e2e/ESIMD/unified_memory_api/atomic_update_usm.cpp

```diff
@@ -0,0 +1,27 @@
+//==------- atomic_update_usm.cpp - DPC++ ESIMD on-device test -------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// RUN: %{build} -DUSE_64_BIT_OFFSET -o %t.out
+// RUN: %{run} %t.out
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include "Inputs/atomic_update.hpp"
+
+int main(void) {
+  queue q(esimd_test::ESIMDSelector, esimd_test::createExceptionHandler());
+
+  auto dev = q.get_device();
+  std::cout << "Running on " << dev.get_info<info::device::name>() << "\n";
+
+  bool passed = true;
+
+  passed &= test_main<false>(q);
```

Comment:
can we have a comment on what false is?

Reply:
it could be:
```suggestion
  constexpr bool TestCacheHintProperties = true;
  passed &= test_main<!TestCacheHintProperties>(q);
```

Reply:
Done in https://github.com/intel/llvm/pull/11833/commits/ffe19702c45355afd8b7ab17d2170cd783e9f670.